networkit/cpp/structures/Partition.cpp

```diff
@@ -134,6 +134,10 @@ std::vector<index> Partition::getVector() const {
     return data;
 }
 
+std::vector<index> Partition::moveVector() {
+    return std::move(this->data);
+}
```

Comment:
I don't think that this is a good idea, it makes the instance of `Partition` useless after it is called. What about returning a const ref?

Reply:
I guess the second copy during coarsening doesn't hurt too bad. I've made the getVector() function return a const ref, and deleted the moveVector() function.